addon/mixins/authenticated-route-mixin.js

```diff
@@ -78,7 +94,12 @@ export default Mixin.create({
       let authenticationRoute = this.get('authenticationRoute');
       assert('The route configured as Configuration.authenticationRoute cannot implement the AuthenticatedRouteMixin mixin as that leads to an infinite transitioning loop!', this.get('routeName') !== authenticationRoute);
 
-      if (!this.get('_isFastBoot')) {
+      if (this.get('_isFastBoot')) {
+        this.get('_cookies').write('ember_simple_auth-redirectTarget', transition.intent.url, {
+          path: `/${authenticationRoute}`,
```

Comment:
`authenticationRoute` will be the Ember route and would contain dots instead of slashes (e.g. `app.internal.login` instead of `app/internal/login`). Is there any way we can get the URL path for the Ember route? If not I think we'll have to set the path to `/`.